src/shader_recompiler/frontend/translate/vector_alu.cpp

```diff
@@ -361,9 +361,11 @@ void Translator::V_BFE_U32(bool is_signed, const GcnInst& inst) {
     SetDst(inst.dst[0], ir.BitFieldExtract(src0, src1, src2, is_signed));
 }
 
-void Translator::V_MAD_I32_I24(const GcnInst& inst) {
-    const IR::U32 src0{ir.BitFieldExtract(GetSrc(inst.src[0]), ir.Imm32(0), ir.Imm32(24), true)};
-    const IR::U32 src1{ir.BitFieldExtract(GetSrc(inst.src[1]), ir.Imm32(0), ir.Imm32(24), true)};
+void Translator::V_MAD_I32_I24(const GcnInst& inst, bool performBitExtract) {
```

Comment:
```suggestion
void Translator::V_MAD_I32_I24(const GcnInst& inst, bool is_signed) {
```
The given name isn't very descriptive of what actually does